Loop/Managers/Alerts/AlertStore.xcdatamodeld/AlertStore.xcdatamodel/contents

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<model type="com.apple.IDECoreDataModeler.DataModel" documentVersion="1.0" lastSavedToolsVersion="15702" systemVersion="19E287" minimumToolsVersion="Automatic" sourceLanguage="Swift" userDefinedModelVersionIdentifier="">
+    <entity name="StoredAlert" representedClassName=".StoredAlert" syncable="YES" codeGenerationType="class">
+        <attribute name="acknowledgedDate" optional="YES" attributeType="Date" usesScalarValueType="NO"/>
+        <attribute name="backgroundContent" optional="YES" attributeType="String"/>
+        <attribute name="foregroundContent" optional="YES" attributeType="String"/>
+        <attribute name="identifier" attributeType="String"/>
+        <attribute name="isCritical" attributeType="Boolean" usesScalarValueType="YES"/>
+        <attribute name="issuedDate" attributeType="Date" usesScalarValueType="NO"/>
+        <attribute name="modificationCounter" attributeType="Integer 64" defaultValueString="0" usesScalarValueType="YES"/>
+        <attribute name="retractedDate" optional="YES" attributeType="Date" usesScalarValueType="NO"/>
+        <attribute name="sound" optional="YES" attributeType="String"/>
+        <attribute name="trigger" optional="YES" attributeType="String"/>
```

Comment:
I was looking for the delay attribute for future scheduled alerts, and couldn't find it, and then realized it must be encoded in the trigger JSON.  Do you see any current need, or a likely future need for querying events based on when they would be issued? If so, maybe consider encoding the Trigger fields flatly in this object.

Reply:
Yeah, after talking with Paul today about how they may want to display the Notifications, I think we may have display policy around future vs. immediate alerts.  I'm going to change this now, to avoid migration later.